src/plugins/controls/public/options_list/components/options_list_popover_action_bar.tsx

```diff
@@ -52,23 +53,20 @@ export const OptionsListPopoverActionBar = ({
     <div className="optionsList__actions">
       <EuiFormRow fullWidth>
         <EuiFlexGroup className="optionsList__searchSortRow" gutterSize="xs" responsive={false}>
```

Comment:
**nit**: We should keep the class name in sync with the changes - since the top row no longer includes the sort button, we should reflect this in the class name

```suggestion
        <EuiFlexGroup className="optionsList__searchRow" gutterSize="xs" responsive={false}>
```